test/validate.js

```diff
@@ -0,0 +1,484 @@
+const tape = require('tape')
+const fs = require('fs')
+const bencode = require('bencode')
+const bfe = require('ssb-bfe')
+const crypto = require('crypto')
+const bb = require('../')
+
+const vec = JSON.parse(
+  fs.readFileSync('test/testvector-metafeed-managment.json', 'utf8')
+)
+
+const badVec = JSON.parse(
+  fs.readFileSync('test/testvector-metafeed-bad-messages.json', 'utf8')
+)
+
+function entryToMsgValue(entry) {
+  const { Author, Sequence, Previous, Timestamp, Signature } = entry
+  let [content, contentSignature] = entry.HighlevelContent
+  if (typeof content.nonce === 'string') {
+    content.nonce = Buffer.from(content.nonce, 'base64')
+  }
+  contentSignature = bfe.decode(Buffer.from(contentSignature.HexString, 'hex'))
+  const signature = bfe.decode(Buffer.from(Signature, 'hex'))
+
+  return {
+    author: Author,
+    sequence: Sequence,
+    previous: Previous,
+    timestamp: Timestamp,
+    content,
+    contentSignature,
+    signature,
+  }
+}
+
+tape('validation works (validateSingle)', function (t) {
+  const msg1 = entryToMsgValue(vec.Entries[0])
+  const msg2 = entryToMsgValue(vec.Entries[1])
+  const msg3 = entryToMsgValue(vec.Entries[2])
+
+  console.log('[ basic tests ]')
```

Comment:
you can use `t.pass()` for printing these kinds of messages that makes the tests easier to read.